docs/howto/custom-management-commands.txt

```diff
@@ -256,6 +256,15 @@ All attributes can be set in your derived class and can be used in
     because attempting to set the locale needs access to settings. This
     condition will generate a :class:`CommandError`.
 
+.. attribute:: BaseCommand.style
+
+    An instance attribute that defines the style of the output, when writing to,
+    e.g., stdout.
+
+    Used for emphasizing messages by adding a color to them. To avoid text
+    higlighting, pass the :djadminopt:`--no-color` option when running your
+    command. To modify the color palette, see the :ref:`syntax-coloring`.
```

Comment:
I am not sure I understood this comment. In fact, I've tried to find some references in the docs about options. I've found options in the code though, in utils/termcolors.py 
There all available options are provided (SUCCESS, WARNING, NOTICE, etc.).